android/KMAPro/build.sh

```diff
@@ -19,23 +14,51 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 . "$KEYMAN_ROOT/resources/build/build-help.inc.sh"
 . "$KEYMAN_ROOT/resources/build/build-download-resources.sh"
 
-echo Build KMAPro
-
-#
-# Prevents 'clear' on exit of mingw64 bash shell
-#
-SHLVL=0
-
-display_usage ( ) {
-    echo "build.sh [-no-daemon] [-debug] [-download-resources]"
-    echo
-    echo "Build Keyman for Android"
-    echo "  -no-daemon              Don't start the Gradle daemon. Use for CI"
-    echo "  -upload-sentry          Uploads debug symbols, etc, to Sentry"
-    echo "  -debug                  Compile only Debug variant"
-    echo "  -download-resources     Download sil_euro_latin.kmp and nrc.en.mtnt.model.kmp from downloads.keyman.com"
-    exit 1
-}
+# This script runs from its own folder
+cd "$THIS_SCRIPT_PATH"
+
+# ################################ Main script ################################
+
+# Definition of global compile constants
+
+CONFIG="release"
+BUILD_FLAGS="build -x lint -x test"                     # Gradle build w/o test
+TEST_FLAGS="-x assembleRelease lintRelease testRelease" # Gradle test w/o build
+DAEMON_FLAG=
+
+builder_describe "Builds Keyman for Android app." \
+  "@../KMEA" \
```

Comment:
```suggestion
  "@/android/KMEA" \
```

For performance reasons use repo-absolute paths (see also #8416)